dataset_builders/hf/sciarg/README.md

```diff
@@ -0,0 +1,227 @@
+# Dataset Card for "SciArg"
+
+## Dataset Description
+
+- **Repository:** [https://github.com/anlausch/ArguminSci](https://github.com/anlausch/ArguminSci)
+
+### Dataset Summary
+
+The SciArg dataset is an extension of the Dr. Inventor corpus (Fisas et al., [2015](https://aclanthology.org/W15-1605.pdf), [2016](https://aclanthology.org/L16-1492.pdf)) with an annotation layer containing
+fine-grained argumentative components and relations, believing that argumentation needs to
+be studied in combination with other rhetorical aspects. It is the first publicly-available argument-annotated corpus of scientific publications (in English), which allows for joint analyses of argumentation and other
+rhetorical dimensions of scientific writing." ([Lauscher et al., 2018](<(https://aclanthology.org/W18-5206/)>), pp. 40-41)
+
+### Supported Tasks and Leaderboards
+
+- **Tasks**: Argumentation Mining, Component Identification, Relation Identification
+- **Leaderboard:** [More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+
+### Languages
+
+The language in the dataset is English (scientific academic publications on computer graphics).
```

Comment:
The sections `Dataset Description`, `Dataset Summary`, `Supported Tasks and Leaderboards`, and `Languages should` go the PIE dateset card.

Reply:
moved from hf/readme to pie/readme in 22be467